### PR TITLE
Add origins to bxt_hud_tas_editor_status

### PR DESCRIPTION
**Pull request by khanghugo**
I think these so far are what needed.

### CODE REVIEW

**Comment by khanghugo** [comment]
This one can wait a bit, I am adding more things and passing an array instead of whatever I am doing here...

File: BunnymodXT/hud_custom.cpp [hunk 1]
```diff
@@ -544,9 +546,18 @@ namespace CustomHud
 			}
 			else
 			{
-				out << "X: " << player.origin[0] << "\n"
-					<< "Y: " << player.origin[1] << "\n"
-					<< "Z: " << player.origin[2];
+				if (frame_bulk_selected)
+				{
+					out << "X: " << frame_bulk_status.xpos << "\n"
```
Comment (YaLTeR) [inline BunnymodXT/hud_custom.cpp:551]: It's kinda weird how all frame bulk info is in the tas editor status hud, but just the origin is suddenly in the origin hud, and not always. I'd rather put the frame bulk origin into the tas editor status hud too.

Comment (khanghugo) [reply BunnymodXT/hud_custom.cpp:551]: Hmm, wouldn't that make the thing bit crowded? I am intending to have bxt_hud_velocity dumped into `frame_bulk_status` as well.

Comment (YaLTeR) [reply BunnymodXT/hud_custom.cpp:551]: Well, yeah, but it's better than this surprising behavior imo

Comment (khanghugo) [reply BunnymodXT/hud_custom.cpp:551]: Oh, never mind, I don't need the bxt_hud_velocity, forgot it is plainly there.

Comment (khanghugo) [reply BunnymodXT/hud_custom.cpp:551]: So this means this change to bxt_hud_origin should be reverted and have the frame bulk origin in bxt_hud_tas_editor_status instead?

Comment (YaLTeR) [reply BunnymodXT/hud_custom.cpp:551]: Yeah

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**